packages/relay/src/lib/services/cacheService/cacheService.ts

```diff
@@ -168,11 +171,11 @@ export class CacheService {
    * @param {string} [requestIdPrefix] - The optional request ID prefix.
    * @returns {Promise<any>} A Promise that resolves with the cached value or null if not found.
    */
-  public getSharedWithFallback(key: string, callingMethod: string, requestIdPrefix?: string): any {
+  public async getSharedWithFallback(key: string, callingMethod: string, requestIdPrefix?: string): Promise<any> {
     if (this.isSharedCacheEnabled) {
-      return this.getAsync(key, callingMethod, requestIdPrefix);
+      return await this.getFromSharedCache(key, callingMethod, requestIdPrefix);
     } else {
-      return this.get(key, callingMethod, requestIdPrefix);
+      return this.getFromInternalCache(key, callingMethod, requestIdPrefix);
```

Comment:
And also change this to:
```
return await this.getFromInternalCache(key, callingMethod, requestIdPrefix);
```

Reply:
Please, also check for other such usages which haven't been updated to use `await` for the `LocalLRUCache`